gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Feb 09 21:49:18 EST 2016
+#Sat Sep 24 14:55:55 EDT 2016
```

Comment:
this needs to be in the git ignore file. Remove it from this pr, and i can add it to the ignore file.